guides/common/modules/con_configuring-an-ldap-server-as-an-external-identity-provider-for-project.adoc

```diff
@@ -1,5 +1,13 @@
 [id="configuring-an-ldap-server-as-an-external-identity-provider-for-project_{context}"]
 = Configuring an LDAP server as an external identity provider for {Project}
+[[Using_LDAP_satellite]]
```

Comment:
unsure about this. We currently don't keep old references.

Reply:
Unfortunately, we link to the `Using_LDAP_satellite` anchor from Satellite web UI. That's why I'd prefer to keep the old ID, as long as it doesn't break anything of course.

Reply:
OK, makes sense! Do you mind adding a comment to explain this?

If I click on it, it shows the correct anchor: https://theforeman-foreman-documentation-preview-pr-2975.surge.sh/nightly/Installing_Server/index-satellite.html#configuring-an-ldap-server-as-an-external-identity-provider-for-project_satellite :heavy_check_mark: 
 (relevant for all other build targets)

Reply:
Sure, I added a comment to explain the purpose of the secondary anchor.

Reply:
Actually, it looks like I could just update the ID in [foreman_theme_satellite](https://github.com/RedHatSatellite/foreman_theme_satellite/blob/b6ee897d49a8b2d9f8b2589e9105ad5b433e5868/lib/foreman_theme_satellite/documentation.rb#L51).

Let's leave things as they are in this PR, but I'll create an issue for this. The plan would be to update the link in foreman_theme_satellite, and then to remove the secondary ID from guides/common/modules/con_configuring-an-ldap-server-as-an-external-identity-provider-for-project.adoc.